guides/common/modules/proc_installing-kernelcare-plugin.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Installing_Foreman_KernelCare_Plug-in_{context}"]
+= Installing Foreman KernelCare Plug-in
+
+Use the following procedure to install KernelCare plug-in.
+After installing KernelCare, you can configure {ProjectServer} to enable live kernel patching on managed hosts.
+
+.Procedure
+. On your {ProjectServer}, install the plug-in package:
++
+[source,none]
+----
+# {package-install} tfm-rubygem-foreman_kernel_care
```

Comment:
The package name is it's EL7-only. For EL8 (the only version we support today) it's:
```suggestion
# {package-install} rubygem-foreman_kernel_care
```

But can we please do this properly by using the installer? I've opened https://projects.theforeman.org/issues/35800 and https://github.com/theforeman/puppet-foreman/pull/1099. After that's merged one should also be done for the installer itself. After that the instruction becomes `{foreman-installer} --enable-foreman-plugin-kernel-care`.

Reply:
FYI: it still needs a PR to foreman-installer itself and I don't have enough time to include that before 3.5.0-rc2 but I can try to make sure it makes it into the final release.

Reply:
I appreciate it! Feel free to ping me or my colleagues for reviews.